app/models/workflow.js

```diff
@@ -5,7 +5,11 @@ export default DS.Model.extend({
   name: DS.attr('string'),
   tag: DS.attr('string'),
   versions: DS.hasMany('workflow-version'),
+  state: DS.attr('string'),
   latestVersion: Ember.computed('versions.[]', function() {
     return this.get('versions.lastObject');
+  }),
+  isActive: Ember.computed('state', function() {
```

Comment:
You can use [Ember.computed.equal()](https://api.emberjs.com/ember/2.13/namespaces/Ember.computed/methods/equal?anchor=equal) here: `isActive: Ember.computed.equal('state','A')`